pulsar-broker/src/main/java/org/apache/pulsar/broker/namespace/NamespaceService.java

```diff
@@ -880,12 +880,22 @@ void splitAndOwnBundleOnceAndRetry(NamespaceBundle bundle,
                                     for (NamespaceBundle sBundle : splittedBundles.getRight()) {
                                         checkNotNull(ownershipCache.tryAcquiringOwnership(sBundle));
                                     }
+                                    updateNamespaceBundlesForPolicies(nsname, splittedBundles.getLeft())
+                                            .exceptionally(e -> {
+                                                String msg = format("failed to update namespace policies [%s], "
+                                                                + "NamespaceBundle: %s due to %s",
+                                                        nsname.toString(), bundle.getBundleRange(), e.getMessage());
+                                                LOG.warn(msg);
+                                                updateFuture.completeExceptionally(
```

Comment:
I wonder what will happen if `updateFuture` completeExceptionally here and complete success in L896(after updateNamespaceBundles success). There is no executing order guarantee for these two ops.

Reply:
> I wonder what will happen if updateFuture completeExceptionally here and complete success in L896(after updateNamespaceBundles success).

Local policies were updated successfully, but policies update failed.

> There is no executing order guarantee for these two ops.

I can add an order guarantee: updateNamespaceBundlesForPolicies is only executed after updateNamespaceBundles is executed successfully.

Reply:
I add an order guarantee: updateNamespaceBundlesForPolicies is only executed after updateNamespaceBundles is executed successfully!

@Jason918 PTAL,thanks!